R/apa_print_glm.R

```diff
@@ -1,60 +1,54 @@
-#' Format statistics (APA 6th edition)
+#' Typeset statistics from GLM
 #'
-#' These methods take [glm][glm()] and [lm][lm()] objects to create formatted character
-#' strings to report the results in accordance with APA manuscript guidelines.
+#' These methods take (general) linear model objects to create formatted
```

Comment:
I think this should be "(generalized) linear model", I would also spell out GLM in the title